CHECKTHIS

```diff
@@ -0,0 +1,53 @@
+# Specific uses in Pig-Latin
+def t(str):
+    #Returns the 2 letters added together to check against
+    return str[0]+str[1]
+
+# Speicfic use characters                
+lst = ['sh', 'gl', 'ch', 'ph', 'tr', 'br', 'fr', 'bl', 'gr', 'st', 'sl', 'cl', 'pl', 'fl']
+# My input section
+sentence = input('Type what you would like translated into pig-latin and press ENTER: ')
+# sentence.split() converts the sentence into a list per word
+sentence = sentence.split()
+for k in range(len(sentence)):
```

Comment:
I think python can iterate over the list directly. So you could do `for word in sentence:`